src/cpp/tests/automation/testthat/test-automation-completions.R

```diff
@@ -27,4 +27,183 @@ test_that("autocompletion doesn't trigger active bindings", {
    output <- remote$consoleOutput()
    expect_false(tail(output, 1) == "[1] \"active\"")
    
+   remote$consoleClear()   
+})
+
+test_that("autocompletion in console produces the expected completion list for new variables", {
+   
+   remote$consoleClear()
+   
+   remote$consoleExecute('
+      foobar <- 42
+      foobaz <- 42
+   ')
+   
+   remote$keyboardExecute("foo", "<Tab>")
+   completionListEl <- remote$jsObjectViaSelector("#rstudio_popup_completions")
+   completionText <- completionListEl$innerText
+   
+   expect_true(grepl("foobar", completionText, fixed = TRUE))
+   expect_true(grepl("foobaz", completionText, fixed = TRUE))
+   
+   remote$consoleClear()   
+})
+
+# WIP
```

Comment:
Is this still WIP? :-)

Reply:
I believe that the proper response is "d'oh!"

Reply:
Do you think that it would make more sense to clear the console at the end, cleaning up after every run? Or leave the output in the console, so that we can see the end state if we're running an individual test?